frontend/react/style-guide.md

```diff
@@ -130,14 +130,19 @@ src
     );
 
     // good
-    function Listing({ hello }) {
+    interface Props {
+      hello: string
```

Comment:
Sorry for the nitpick, since we've linter checks for missing ";" we should add them on interface fields too
```suggestion
      hello: string;
```